src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -79,13 +89,70 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     String user = userService.getCurrentUser().getEmail();
+    // Get the message entered by the user.
     String userText =
         Jsoup.clean(request.getParameter("text"), Whitelist.none());
-    String replacement = "<img src=\"$1\" />";
-    String textWithImagesReplaced = userText.replaceAll(REGEX, replacement);
+    /*   String replacement = "<img src=\"$1\" />";
```

Comment:
should this be commented out?? (if so, plz ignore sorry lmaooo)

Reply:
took it out!